src/features/creators/creationHelpers.ts

```diff
@@ -0,0 +1,129 @@
+    }
+    return copilotChoice === 'Yes';
+}
+
+export async function removeCopilotInstructions(destFolder: string) {
+    const copilotFolder = path.join(destFolder, '.github');
+    if (await fs.pathExists(copilotFolder)) {
+        await fs.remove(copilotFolder);
+    }
+}
+
+export async function quickCreateNewVenv(envManagers: EnvironmentManagers, destFolder: string) {
+    try {
+        // get the environment manager for venv
+        const envManager: EnvironmentManager | undefined = envManagers.managers.find(
+            (m) => m.id === 'ms-python.python:venv',
+        );
+        const destUri = Uri.parse(destFolder);
+        if (envManager && envManager.create) {
+            // with quickCreate enabled, user will not be prompted when creating the environment
+            const options: CreateEnvironmentOptions = { quickCreate: false };
+            if (envManager.quickCreateConfig) {
+                options.quickCreate = true;
+            }
+            const pyEnv = await envManager.create(destUri, options);
+            // comes back as undefined if this doesn't work
+            traceVerbose(`Created venv at: ${pyEnv?.environmentPath} using ${envManager.name}`);
+        } else {
+            // find an environment manager that supports create
+            const envManager = envManagers.managers.find((m) => m.create);
+            if (envManager) {
+                const pyEnv = await envManager.create(destUri, {});
+                traceVerbose(`Created venv at: ${pyEnv?.environmentPath} using ${envManager.name}`);
+            }
+            // If no environment manager supports create, show an error message
+            window.showErrorMessage(
+                `No environment manager found that supports creating a new environment, skipping...`,
+            );
+        }
+    } catch (err) {
+        window.showErrorMessage(`Failed to create virtual environment: ${err}`);
+    }
+}
+
+/**
+ * Replaces all occurrences of a string in a single file's contents, handling special regex characters in the search value.
+ * @param filePath The path to the file to update.
+ * @param searchValue The string to search for (will be escaped for regex).
+ * @param replaceValue The string to replace with.
+ */
+export async function replaceInFile(filePath: string, searchValue: string, replaceValue: string) {
+    // Escape special regex characters in searchValue
+    const escapedSearchValue = searchValue.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+    const regex = new RegExp(escapedSearchValue, 'g');
+    let content = await fs.readFile(filePath, 'utf8');
+    if (content.includes(searchValue)) {
+        content = content.replace(regex, replaceValue);
+        await fs.writeFile(filePath, content, 'utf8');
+    }
+}
+
+// Helper to recursively replace all occurrences of a string in file/folder names and file contents
+export async function replaceInFilesAndNames(dir: string, searchValue: string, replaceValue: string) {
```

Comment:
@karthiknadig is it the correct strategy here to copy then rename or should I not do the rename in place like this?